src/sentry/web/frontend/react_page.py

```diff
@@ -29,6 +31,9 @@ def handle_react(self, request):
 # generic
 class ReactPageView(OrganizationView, ReactMixin):
     def handle(self, request, **kwargs):
+        if 'project_id' in kwargs and request.GET.get('onboarding'):
+            project = Project.objects.filter(slug=kwargs['project_id']).first()
```

Comment:
This won't be unique at all and very likely to yield the wrong project. You also need to scope it with organization.

Since `ReactPageView` is inheriting from `OrganizationView`, that means we have the organization as a kwarg already.

But what you technically want is `ProjectView`: https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/base.py#L404-L428

This would give you project already.